website/src/components/ReviewPage/SequencesDialog.tsx

```diff
@@ -0,0 +1,84 @@
+import { type FC, useState } from 'react';
+
+import { type SequenceEntryToEdit } from '../../types/backend.ts';
+import { BoxWithTabsBox, BoxWithTabsTab, BoxWithTabsTabBar } from '../common/BoxWithTabs.tsx';
+import { FixedLengthTextViewer } from '../common/FixedLengthTextViewer.tsx';
+
+type SequencesDialogProps = {
+    isOpen: boolean;
+    onClose: () => void;
+    dataToView: SequenceEntryToEdit | undefined;
+};
+
+type ProcessedSequence = {
+    label: string;
+    sequence: string | null;
+};
+
+export const SequencesDialog: FC<SequencesDialogProps> = ({ isOpen, onClose, dataToView }) => {
+    const [activeTab, setActiveTab] = useState(0);
+
+    if (!isOpen || !dataToView) return null;
+
+    const processedSequences = extractProcessedSequences(dataToView);
+
+    if (processedSequences.length === 0) {
+        return null;
+    }
+
+    return (
+        <div className='fixed inset-0 flex items-center justify-center z-50 overflow-auto bg-black bg-opacity-30'>
```

Comment:
I think it'd be good to reuse the `BaseDialog.tsx` - then we have the same opacity, close button etc everywhere.

Reply:
I'm not sure if we can style the width of that exactly as here. Happy for anyone to look into it in a follow up but IMO not blocking?

Reply:
yes not blocking